tests/lib/spsc_pbuf/src/main.c

```diff
@@ -5,45 +5,45 @@
  */
 
 #include <ztest.h>
-#include <zephyr/ipc/ipc_icmsg_buf.h>
+#include <zephyr/sys/spsc_pbuf.h>
 
 /* The buffer size itself would be 199 bytes.
- * 212 - sizeof(struct icmsg_buf) - 1 = 199.
+ * 212 - sizeof(struct spsc_pbuf) - 1 = 199.
  * -1 because internal rd/wr_idx is reserved to mean the buffer is empty.
  */
 static uint8_t memory_area[212] __aligned(4);
 
-static void test_icmsg_buf_ut(void)
+static void test_spsc_pbuf_ut(void)
 {
 	static uint8_t rbuf[198];
 	static uint8_t message[20] = {'a'};
-	struct icmsg_buf *ib;
+	struct spsc_pbuf *ib;
```

Comment:
Please also change ib -> pbuf (packet buffer) or similar. All across the file and in similar places.

Reply:
done, changed to `pb`